hs_core/management/commands/update_crossref_metadata_deposit.py

```diff
@@ -0,0 +1,44 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand, CommandError
+
+from hs_core.enums import CrossRefSubmissionStatus
+from hs_core.hydroshare import (deposit_res_metadata_with_crossref,
+                                get_resource_doi)
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+
+
+class Command(BaseCommand):
+    help = "Update crossref metadata deposit for a published resource"
+
+    def add_arguments(self, parser):
+
+        # ID of a published resource for which crossref metadata needs to be updated
+        parser.add_argument('resource_id', type=str, help=('Required. The existing id (short_id) of'
+                                                           ' the published resource'))
+
+    def handle(self, *args, **options):
+
+        if not options['resource_id']:
+            raise CommandError('resource_id argument is required')
+        res_id = options['resource_id']
+        try:
+            resource = get_resource_by_shortkey(res_id, or_404=False)
```

Comment:
do you need `resource = resource.get_content_model()`

Reply:
This function `get_resource_by_shortkey` already does that.

Reply:
nice thanks